js/adapt-hint.js

```diff
@@ -1,6 +1,7 @@
 define([
-  'core/js/adapt'
-], function (Adapt) {
+  'core/js/adapt',
+  'core/js/reactHelpers'
```

Comment:
You shouldn't need to import this here unless you're calling it in this is file. `compile` is also a handlebars helper, two functions, same name, same behaviour, different use-cases. This one is to compile handlebars in jsx Files.